File: domain-api/src/main/kotlin/com/bookmark/model/Group.kt
package com.bookmark.model

import java.time.LocalDateTime

data class Group(
        val groupId: Long? = null,
        val groupName: String? = null,
        val groupContextName: String? = null,
        val groupContext: GroupContext? = null,
        val groupUrl: String? = null,
        val createdOn: LocalDateTime? = null,
        val users: List<GroupUser>? = emptyList(),
        val urls: List<GroupUrl>? = emptyList()
)

File: bookmark-ui/src/app/shared/model/group.ts
import { GroupUser } from './group-user';
import { GroupUrl } from './group-url';

export interface Group{
     groupId: number;
     groupName: string;
     groupContextName: string;
     groupContext: string;
     groupUrl: string;
     createdOn: string;
     users: GroupUser[];
     urls: GroupUrl[];
}
